printing/proof_diffs.ml

```diff
@@ -342,14 +340,14 @@ let hyp_list_to_pp hyps =
 
 let unwrap g_s =
   match g_s with
-  | Some (goal, sigma) -> goal_info goal sigma
+  | Some g_s -> goal_info g_s
   | None -> ([], CString.Map.empty, Pp.mt ())
 
-let diff_goal_ide og_s ng nsigma =
-  diff_goal_info (unwrap og_s) (goal_info ng nsigma)
+let diff_goal_ide og_s ng =
+  diff_goal_info (unwrap og_s) (goal_info ng)
 
-let diff_goal ?og_s ng ns =
-  let (hyps_pp_list, concl_pp) = diff_goal_info (unwrap og_s) (goal_info ng ns) in
+let diff_goal ?og_s ng =
```

Comment:
Some time ago, IIRC, Maxime suggested that the interface between `proof_diffs` and `printer` was overly complex.  He's probably right.  Proof diffs could have formatted the old and new contexts as `Pp.t`s and then zipped them together afterwards.